scalyr_agent/third_party/tcollector/collectors/0/procstats.py

```diff
@@ -66,36 +72,36 @@ def print_numa_stats(numafiles):
                           # miss: process wanted another node and got it from
                           # this one instead.
                           ("numa_miss", "miss")):
-            print ("sys.numa.zoneallocs %d %s node=%d type=%s"
-                   % (ts, stats[stat], node_id, tag))
+            print(("sys.numa.zoneallocs %d %s node=%d type=%s"
+                   % (ts, stats[stat], node_id, tag)))
```

Comment:
Do we need the double parens here?

Reply:
No, I've missed those after modernize, again... Thanks.